src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data_tab.cljs

```diff
@@ -65,7 +65,17 @@
           :onChange #(let [value (common/get-text refs "filter")
                            entities (get-in props [:entity-map value])]
                       (swap! state assoc :entities entities :entity-type value))}
-         (keys (:entity-map props)))]
+         (keys (:entity-map props)))
+       (let [selected-entity-type (cond
+                                    (:entity-type @state) (:entity-type @state)
+                                    (:entity-map props) (first (keys (:entity-map props))))]
+         (when-not (nil? selected-entity-type)
+           [:span {} " "
+            [:a {:style {:textDecoration "none"}
```

Comment:
I think a `:marginLeft "1ex"` here would be slightly cleaner than a `span` with a space in it.